docs/api.md

```diff
@@ -60,8 +60,8 @@ see [`mozilla/fxa-js-client`](https://github.com/mozilla/fxa-js-client).
     * [POST /session/verify/recoveryCode (:lock: sessionToken)](#post-sessionverifyrecoverycode)
   * [Recovery key](#recovery-key)
     * [POST /recoveryKey (:lock: sessionToken)](#post-recoverykey)
-    * [GET /recoveryKey/:recoveryKeyId (:lock: accountResetToken)](#get-recoverykey)
-    * [POST /recoveryKey/exists (:lock: sessionToken)](#post-recoverykeyexists)
+    * [GET /recoveryKey/{recoveryKeyId} (:lock: accountResetToken)](#get-recoverykeyrecoverykeyid)
+    * [POST /recoveryKey/exists (:lock::unlock: sessionToken)](#post-recoverykeyexists)
```

Comment:
Is this supposed to be both :lock: and :unlock:?

Reply:
Yep, that's how we indicate optional authentication strategies:

https://github.com/mozilla/fxa-auth-server/blob/master/lib/routes/recovery-key.js#L128